net-proxy/3proxy/3proxy-0.8.9.ebuild

```diff
@@ -8,7 +8,7 @@ HOMEPAGE="http://www.3proxy.ru/"
 SRC_URI="https://github.com/z3APA3A/3proxy/archive/${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~x86 ~amd64 ~ppc"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~m68k ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-fbsd"
```

Comment:
I've checked package's home page, and it says:

> Currently 3proxy is tested to work under Windows since 98 (Windows before Vista reqires lite version) and up to latest version both i386 and x64, FreeBSD/i386/x64, NetBSD/i386/x64, OpenBSD/i386/x64, Linux/i386/x64/PPC/Alpha/arm, Mac OS X, Solaris/i386/x64.

so looks like we need to change KEYWORDS. Can you please give me a link where I can read how to initiate "keyword request" process?